wordpress-mobile/gutenberg-mobile
# Remove left&right borders from focused block in portrait mode

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/424

There's some slight but noticeable movement when a block changes focus in portrait mode.

Before:
![ezgif com-video-to-gif 6](https://user-images.githubusercontent.com/5032900/50353617-d9740b80-0559-11e9-98a5-3f021072643e.gif)

After:
![ezgif com-video-to-gif 5](https://user-images.githubusercontent.com/5032900/50353095-32db3b00-0558-11e9-9cf8-9bca670dff9e.gif)

**To Test**

Portrait:

- Switch focus between 2 blocks and verify that there's no change in block content size. 

Landscape:

- Open the app in iPhone X & Landscape mode
- Verfiy that focused block has left&right borders

iPad

- Open the app in iPad & Portrait
- Verfiy that focused block has left&right borders

- Open the app in iPad & Landscape
- Verfiy that focused block has left&right borders
